frontend/test/__support__/e2e/helpers/e2e-boolean-helpers.js

```diff
@@ -0,0 +1,24 @@
+// until we have a test dataset that includes boolean data, we can use this questions to test booleans
+const BOOLEAN_QUERY =
+  'select 0::integer as "integer", true::boolean AS "boolean" union all \nselect 1::integer as "integer", false::boolean AS "boolean" union all \nselect null as "integer", true::boolean AS "boolean" union all \nselect -1::integer as "integer", null AS "boolean"';
+
+export const setupBooleanQuery = () => {
+  cy.intercept("POST", "/api/dataset").as("dataset");
+
+  cy.createNativeQuestion(
+    {
+      name: "16386",
```

Comment:
```suggestion
      name: "Boolean Query",
```

Let's lose the reference to the specific issue if the idea is to reuse this helper.

Reply:
Alternatively, we can pass the `name` parameter to the function and set the default value to "Boolean Query".